TimJohnson/Homework_Week1.js

```diff
@@ -0,0 +1,139 @@
+1a)	"1" == 1; 	true;  using == means equal to and will cast the 1 on the right side of the equation as a string
+1b)	"1" === 1;  false;  using === means that the comparison includes not only equal value but equal type.  (here is string versus number)
+1c)	 x == 'x';	false;  string 'x' is converted tonumber(x) which is different than the number x
```

Comment:
Points for false. In this situation, however, what is being evaluated is variable x versus string x. Unless variable x contains as its value 'x', every comparison will return false. Nothing is being converted in this case.